acme/acme/standalone.py

```diff
@@ -22,10 +23,40 @@
 # pylint: disable=too-few-public-methods,no-init
 
 
+def _ipv6_enabled(port):
```

Comment:
Given that listening on "::" will still work with subsystem disabled in Linux (at least for local connections over ipv4), this is probably not strictly necessary. However, listening on "::1" would bomb, and this ensures that ipv6 is enabled, so I think it's good.